packages/vue-components/src/Modal.vue

```diff
@@ -131,6 +130,10 @@ export default {
         overflow: auto;
     }
 
+    .max-height {
+        max-height: 90vh;
```

Comment:
Personally I think its better to have the max height as 100vh to be consistent with previous display

Reply:
The thing is that when you scroll to the bottom, there is this nice gap for the previous display
![image](https://github.com/MarkBind/markbind/assets/55905659/6701b24b-4cc8-409f-97ec-486699e80dc4)

But will be absent with `100vh`
![image](https://github.com/MarkBind/markbind/assets/55905659/3ec42190-8d81-4665-ae81-65d8ea9e42ad)

Will that still be ok?

Reply:
oh I understand where you are coming from. Ill still prefer the 100vh though, I think the gap below looks a little awkward with 90